package.json

```diff
@@ -12,6 +12,7 @@
     "prisma:generate": "npx prisma generate",
     "prisma:validate": "npx prisma validate",
     "migrate:postgres": "prisma migrate dev",
+    "migrate:dev": "dotenv -e .env.local -- npx prisma migrate dev",
```

Comment:
Why is this needed?

Reply:
I tried to load the env to run the workflows , it doesn't work , we need to add secrets to GitHub in order for that to work

Reply:
Removed this line